locallib.php

```diff
@@ -454,7 +474,11 @@ function moodleoverflow_get_post_full($postid) {
     $params = array();
     $params['postid'] = $postid;
 
-    return $DB->get_record_sql($sql, $params);
+    $post =  $DB->get_record_sql($sql, $params);
+    if($post->userid === 0) {
+        $post->message = get_string('privacy:anonym_post_name','mod_moodleoverflow');
```

Comment:
I think referring to this as `_name` is confusing, seeing that this is actually the post content.